Makefile

```diff
@@ -68,7 +68,7 @@ oci-image-tool:
 
 schema-fs:
 	@echo "generating schema fs"
-	@cd schema && echo -e "$$(cat ../.header)\n\n$$(go generate)" > fs.go
+	@cd schema && echo "$$(cat ../.header)\n\n$$(go generate)" > fs.go
```

Comment:
`-e` isn't supported on BSD userspace tools. Removing it didn't change the output.